### ISSUE TITLE
Can't play after saving in custom location (Multiplayer and Singleplayer)

### ISSUE THREAD

**Issue by nevth** [yairm210/Unciv#10353]
**Platform**
Android 12

**Version**
4.8.14 [Latest](https://github.com/yairm210/Unciv/releases/latest)

**Describe the bug**
I wanted to load from custom location and it will show "waiting for civ" while i play that specific civ.

**To Reproduce**
1. Save game to custom location
2. Load said file
3. You cant press any buttons, the game still tells "your city etc..." but everything is greyed out.

```

```
</details>

**Expected behavior**
1. Save game to custom location
2. Load said file
3. Interact with your civ.

**Screenshots**
Here is the autosave ss and the custom location ss:

![signal-2023-10-25-082734_002](https://github.com/yairm210/Unciv/assets/80281524/89655a19-d6ee-4a17-add1-25244c5d3857)

![signal-2023-10-25-082731_002](https://github.com/yairm210/Unciv/assets/80281524/b1a8d986-1699-455e-974e-7761c02d7701)

**Comment by yairm210** [comment]
We should block this from happening, not fix

**Comment by CrsiX** [comment]
TL;DR
I would vote for _do not allow saving MP games locally/to custom location_.

> Any mp specialist still here? @CrsiX ?

I can say that this should be possible with V2, if you use a custom client, too. At least for now. It also applies to V1, where this could be used to reset game states to a previous state if I understand the issue correctly, theoretically. Speaking of which, sorry for being inactive on the V2 for so long :/

Actually, saving a MP game locally (or to custom location, should be treated the same IMHO, is it maybe the same?) should either be not allowed at all (the server has the legitimate version, not the client, so load it from there instead from local source?) or should be converted to a local-only game where all other civilizations are removed, so that you can't see what the other civs would be doing (to prevent exploiting that easily)? Of course, option 1 is much easier to do; also option 2 is probably not so easy to understand as a player ("where did all the other civs go?") and still opens up the current MP map to explore locally.

**Comment by nevth** [comment]
> Huh, didn't know there actually is a "Give Gold" quest...
> 
> "Save to custom location" was never meant to be used with multiplayer games. The behaviour you're showing is not OK, but the question is - whether we try to fix a cause or better block the feature because that may open multiplayer exploits (considering the hotseat variant, where just maybe remotely possible someone may use it for play-by-mail???) Any mp specialist still here? @CrsiX ?
> 
> Also - I see another font glitch on Android 12 - which font did you choose in Settings-Advanced?

I have a fold 4.

Here is the setting Screenshot.

Also  custom saving is not working in singleplayer mode (as shown above)

**Comment by nevth** [comment]
![Uploading Screenshot_20231026_213023_Unciv.jpg…]()

**Comment by SeventhM** [comment]
Closed as wontfix, and solved as per #10358
